packages/suite-web-landing/pages/index.tsx

```diff
@@ -70,13 +76,23 @@ const Index = () => {
                     <Wrapper>
                         <StyledHeroCta>
                             <StyledHeadline>
-                                <Translation
-                                    id="TR_SUITE_WEB_LANDING_HEADLINE"
-                                    values={{
-                                        em: chunks => <em>{chunks}</em>,
-                                        lineBreak: <br />,
-                                    }}
-                                />
+                                {isFromMytrezor ? (
+                                    <Translation
+                                        id="TR_SUITE_WEB_LANDING_HEADLINE_FROM_MYTREZOR"
+                                        values={{
+                                            em: chunks => <em>{chunks}</em>,
+                                            lineBreak: <br />,
+                                        }}
+                                    />
+                                ) : (
+                                    <Translation
+                                        id="TR_SUITE_WEB_LANDING_HEADLINE"
+                                        values={{
+                                            em: chunks => <em>{chunks}</em>,
+                                            lineBreak: <br />,
+                                        }}
+                                    />
+                                )}
```

Comment:
One last comment get lost somewhere: to keep it DRY, you could write just:
```suggestion
                               <Translation
                                    id={
                                        isFromMytrezor
                                            ? 'TR_SUITE_WEB_LANDING_HEADLINE_FROM_MYTREZOR'
                                            : 'TR_SUITE_WEB_LANDING_HEADLINE'
                                    }
                                    values={{
                                        em: chunks => <em>{chunks}</em>,
                                        lineBreak: <br />,
                                    }}
                                />
```

Reply:
fixed in 1bad78e also with `border-radius`